src/task2.py

```diff
@@ -0,0 +1,35 @@
+"""
+Создайте декоратор, который хранит результаты вызовы функции
+(за все время вызовов, не только текущий запуск программы)
+
+"""
+from datetime import datetime
+
+
+def decorator(func):
```

Comment:
your decorator doesn't return function result - it can confuse user. it should only add logging